lib/kernel-manager.js

```diff
@@ -101,11 +100,7 @@ class KernelManager {
         break;
     }
 
-    if (projectPath != null) {
-      kernelStartDir = projectPath;
-    } else {
-      kernelStartDir = currentPath;
-    }
+    const kernelStartDir = projectPath != null ? projectPath : currentPath;
```

Comment:
whoa it's kinda crazy you can do `''+null` and end up with `'null'`. but I guess the above is kind of exploiting javascript's `null`-handling a bit in the first place.